code/modules/mob/living/carbon/human/species/plasmaman.dm

```diff
@@ -6,21 +6,19 @@
 	dangerous_existence = TRUE //So so much
 	//language = "Clatter"
 
-	species_traits = list(IS_WHITELISTED, NO_BLOOD, NOTRANSSTING)
+	species_traits = list(IS_WHITELISTED, RADIMMUNE, NO_BLOOD, NOTRANSSTING)
 	forced_heartattack = TRUE // Plasmamen have no blood, but they should still get heart-attacks
 	skinned_type = /obj/item/stack/sheet/mineral/plasma // We're low on plasma, R&D! *eyes plasmaman co-worker intently*
 	dietflags = DIET_OMNI
 	reagent_tag = PROCESS_ORG
 
-	//default_mutations=list(SKELETON) // This screws things up
-
 	butt_sprite = "plasma"
 
 	breathid = "tox"
 
-	heat_level_1 = 350  // Heat damage level 1 above this point.
-	heat_level_2 = 400  // Heat damage level 2 above this point.
-	heat_level_3 = 500  // Heat damage level 3 above this point.
+	burn_mod = 1.5
```

Comment:
these are all IPC levels of damage. Do put the exact numbers in the CL please since it's not just a bit.